airflow/data/agencies.yml

```diff
@@ -1110,9 +1110,9 @@ san-joaquin-regional-transit-district:
   agency_name: San Joaquin Regional Transit District
   feeds:
     - gtfs_schedule_url: http://sanjoaquinrtd.com/RTD-GTFS/RTD-GTFS.zip
-      gtfs_rt_vehicle_positions_url: null
-      gtfs_rt_service_alerts_url: null
-      gtfs_rt_trip_updates_url: null
+      gtfs_rt_vehicle_positions_url: https://api.goswift.ly/real-time/san-joaquin/gtfs-rt-vehicle-positions
+      gtfs_rt_service_alerts_url: https://api.goswift.ly/real-time/san-joaquin/gtfs-rt-alerts
```

Comment:
SJRTD only uses Swiftly for VP and TU. According to their [website](https://sanjoaquinrtd.com/developer-portal/), this is their Alerts URL we should be validating https://sanjoaquinrtd.com/rideralerts/RTDrideralerts.xml